### PR TITLE
kvm-host module doc and changes

### PR DESCRIPTION
**Pull request by adean20csc**
KVM-Host module doc

Changes to other module docs and related files

### CODE REVIEW

File: docsite/source/overview.rst [hunk 1]
```diff
@@ -1,29 +1,99 @@
...
+.. code-block:: yaml
+
+  [group1]
+  host1
+  host2
+
+  [group2]
+  host3
+
+  [both_groups]
```
Comment (mtnbikenc) [inline docsite/source/overview.rst]: Syntax should be `[both_groups:children]`

**Comment by mtnbikenc** [comment]
There are a number of unresolved rebases in this PR, `<<<<`.  These should be corrected and the branch rebased against the current csc/master.

**Comment by mtnbikenc** [comment]
Review 👍

**Comment by jcpowermac** [comment]
Review 👍

**Pull request merged**